UK/Area/London TC/TC Bandbox.prf

```diff
@@ -47,6 +47,7 @@ Plugins	Plugin3	\..\..\Data\Plugin\VFPC.dll
 Plugins	Plugin4	\..\..\Data\Plugin\VCH.dll
 Plugins	Plugin5	\..\..\Data\Plugin\TopSky_NODE\TopSky.dll
 Plugins	Plugin5Display0	Standard ES radar screen
+Plugins	Plugin6	\..\..\Data\Plugin\CCDS-R\CCDS-R.dll
```

Comment:
Missing Plugindisplay lines, or does it need to be allowed to draw?

Reply:
Nah only tag items, doesn't need to draw on the display